CONTRIBUTING.md

```diff
@@ -258,8 +258,7 @@ Lab branches should be deleted once they become obsolete - when that is the case
 
 While it is nice to have each individual commit pass the build, this is not a requirement - it is the contributor's branch to play on.
 
-As a general rule, the style and formatting of commit messages should follow the [guidelines for good Git commit messages](http://chris.beams.io/posts/git-commit/).
-Because of the noise it generates on the issue, please do _not_ mention the issue number in the message.
+See section [Commit Message](#commit-message) for how the commit message should look like.
```

Comment:
Have removed this and referred to the "Commit Message" section to reduce duplication.

Reply:
Good change!

However, elsewhere (e.g. `[_Commit Message_](#commit-message)` in the "Communication" section) we use italic text to highlight references to internal sections. Would you mind doing this here, too?

```text
See section [_Commit Message_](#commit-message) for how the commit message should look like.
```